Presentation.Web/app/components/local-config/localOptionList/localOptionList.directive.ts

```diff
@@ -7,7 +7,8 @@
                 editState: "@state",
                 dirId: "@",
                 optionType: "@",
-                currentOrgId: "@"
+                currentOrgId: "@",
```

Comment:
we should do the same for the globaOptionList.directive since if it is not used, it should not be available.. Same choice types affected and same "hide" strategy